Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN hugo \
       --gc \
       --minify \
       --source src \
-      --path-warnings \
+      --printPathWarnings \
+      --printUnusedTemplates \
```

Comment:
- `--path-warnings` got renamed to `--printPathWarnings`
- `--printUnusedTemplates` is a new flag that makes sense to use for us